### ISSUE TITLE
Unexpected token Token(DECIMAL, '0') when loading terraform file 

### ISSUE THREAD

**Issue by Ozymandiiaz** [amplify-education/python-hcl2#72]
Hello,

I have this error when trying to load a terraform output file : 
```bash
Unexpected token Token(DECIMAL, '0') at line 3, column 75.
Expected one of:
        * __ANON_3
```

My terraform output file 
```hcl
output "webapp_site_credential_user_name" {
  description = "Credentials of Webapp"
  value       = join("", azurerm_app_service.appservice.*.site_credential.0.username)
}
```
It's seems the hcl2.load(file) does not like the "site_credential.0" even if this code is working in terraform.
How can i fix this without changing my terraform code ?

**Comment by M-Rushabh** [comment]
#####################################################################
##
##      Terraform module to create web tier for web-tier
##
#####################################################################

output "instances_ips" {
  value = join(",", concat(ibm_is_instance.instance_az1.*.primary_network_interface.0.primary_ipv4_address,ibm_is_instance.instance_az2.*.primary_network_interface.0.primary_ipv4_address))
}

output "instances_names" {
  value = concat(ibm_is_instance.instance_az1.*.name, ibm_is_instance.instance_az2.*.name)
}


Getting same issue for above block in terraform
Can some one please look into it

**Comment by M-Rushabh** [comment]
@aoskotsky-amplify

**Comment by aoskotsky-amplify** [comment]
I don't believe that is valid hcl2. You should be doing `join("", azurerm_app_service.appservice.*.site_credential[0].username)`. See https://github.com/hashicorp/hcl/blob/main/hclsyntax/spec.md#attribute-access-operator
